relayer-cli/src/commands/tx/connection.rs

```diff
@@ -22,12 +24,44 @@ pub struct TxRawConnInitCmd {
     #[options(free, help = "identifier of the source connection")]
     src_connection_id: Option<String>,
 
-    #[options(help = "identifier of the destination connection", short = "d")]
+    #[options(free, help = "identifier of the destination connection")]
```

Comment:
I believe this is optional no ? Maybe if there's a new requirement that if the value is not specified then it is retrieved from somewhere, but that would be a separate issue.

Reply:
In master you need to put `-d <dest_conn_id>` and in help it doesn't show as positional arg but a flag. I think it works in your branch for this particular command because this is the only optional arg (can be omitted) and it's the last one. Otherwise with `free` you can't implement optional params i think. 
Looking through the code I am wondering why we have `Option<>` for `free` params, it doesn't make sense. Anyway, let's leave it as is and will open another issue to fix all CLIs.

Reply:
Agree, the gumdrop documentation is not very clear. We can open a separate issue